<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>781</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup/icons/poitagger.png</normaloff>../../../../.designer/backup/icons/poitagger.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionJpg_export"/>
    <addaction name="actionPng_export"/>
    <addaction name="actionConvert_folder_to_jpg"/>
    <addaction name="actionCreate_subimages"/>
    <addaction name="actionDrucken"/>
    <addaction name="actionDruckansicht"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionEinstellungen"/>
    <addaction name="actionSettingsfile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QDockWidget" name="PixeltempDockWidget">
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9"/>
  </widget>
  <widget class="QDockWidget" name="VerzeichnisDockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>400</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Verzeichnisbaum</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7"/>
  </widget>
  <widget class="QDockWidget" name="ConsoleDockWidget">
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>80</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8"/>
  </widget>
  <widget class="QDockWidget" name="InfoDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>42</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>260</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Exif</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_17">
    <widget class="QDockWidget" name="FMInfoDockWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>280</width>
       <height>84</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>42</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>260</width>
       <height>300</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>Flight</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_18"/>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="PoisDockWidget">
   <property name="windowTitle">
    <string>Pois</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSD_Karte_einlesen"/>
   <addaction name="actionGpx_to_gps"/>
  </widget>
  <widget class="QDockWidget" name="GeoViewDockWidget">
   <property name="windowTitle">
    <string>Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="GpxDockWidget">
   <property name="windowTitle">
    <string>Gpx</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionUAV_Pose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/drone-512.png</normaloff>icons/drone-512.png</iconset>
   </property>
   <property name="text">
    <string>UAV-Pose</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionGPS_Start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GPS-Start</string>
   </property>
  </action>
  <action name="actionFirmware">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Firmware</string>
   </property>
  </action>
  <action name="actionAsctec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Asctec</string>
   </property>
  </action>
  <action name="actionSD_Karte_einlesen">
   <property name="icon">
    <iconset>
     <normaloff>icons/micro_sd-512.png</normaloff>icons/micro_sd-512.png</iconset>
   </property>
   <property name="text">
    <string>SD Karte einlesen</string>
   </property>
  </action>
  <action name="actionJpg_export">
   <property name="text">
    <string>jpg export</string>
   </property>
  </action>
  <action name="actionGoogleMaps">
   <property name="icon">
    <iconset>
     <normaloff>icons/world_marker.jpg</normaloff>icons/world_marker.jpg</iconset>
   </property>
   <property name="text">
    <string>googleMaps</string>
   </property>
  </action>
  <action name="actionPng_export">
   <property name="text">
    <string>png export</string>
   </property>
  </action>
  <action name="actionGpx_to_gps">
   <property name="icon">
    <iconset>
     <normaloff>icons/handheld-gps-device.jpg</normaloff>icons/handheld-gps-device.jpg</iconset>
   </property>
   <property name="text">
    <string>gpx_to_gps</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>allgemein</string>
   </property>
  </action>
  <action name="actionSettingsfile">
   <property name="text">
    <string>Settingsfile</string>
   </property>
  </action>
  <action name="actionConvert_folder_to_jpg">
   <property name="text">
    <string>convert folder to jpg</string>
   </property>
  </action>
  <action name="actionCreate_subimages">
   <property name="text">
    <string>create subimages</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="icon">
    <iconset>
     <normaloff>icons/Printer-icon.png</normaloff>icons/Printer-icon.png</iconset>
   </property>
   <property name="text">
    <string>Drucken (Strg+p)</string>
   </property>
   <property name="toolTip">
    <string>Drucken (Strg+p)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDruckansicht">
   <property name="icon">
    <iconset>
     <normaloff>icons/Preview-icon.png</normaloff>icons/Preview-icon.png</iconset>
   </property>
   <property name="text">
    <string>Druckansicht</string>
   </property>
  </action>
  <zorder>InfoDockWidget</zorder>
  <zorder>PixeltempDockWidget</zorder>
  <zorder>GeoViewDockWidget</zorder>
  <zorder>GpxDockWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
